hw/ip/otbn/dv/otbnsim/sim/sim.py

```diff
@@ -253,6 +253,20 @@ def _step_exec(self, verbose: bool) -> StepRes:
             self.state.take_injected_err_bits()
             return (None, self._on_stall(verbose, fetch_next=True))
 
+        # If there is an RMA request, we treat it a bit like a fatal error, and
+        # abort any instruction that's currently running
+        if self.state.rma_req == LcTx.ON:
+            # TODO: Passing a nonzero value in ErrBits is incorrect: we don't
```

Comment:
Is it worth tagging this todo with a issue number, or do you have a fix in mind for this already?

Reply:
Probably not worth creating a proper issue: I'm actively working on sorting this area out already :-) But thanks for noticing!